sdk/workloads/azure-resourcemanager-workloads/src/test/java/com/azure/resourcemanager/workloads/WorkloadsManagerTests.java

```diff
@@ -129,13 +131,15 @@ public void testCreateMonitor() {
                         .withAddressSpace("172.29.0.0/16")
                         .withSubnet("default", "172.29.0.0/24")
                         .create()
+                        .subnets()
+                        .get("default")
```

Comment:
?

Reply:
The `.withMonitorSubnet` needs to specify `subnet`, not network.